### ISSUE TITLE
pip install fails, does not find readme

### ISSUE THREAD

**Issue by setop** [rmit-ir/KrovetzStemmer#4]
Trying last release : 0.7.
```
$ pip install KrovetzStemmer
Collecting KrovetzStemmer
  Using cached https://files.pythonhosted.org/packages/46/37/2a99dd9738799a1d0ef69483a415a29ebf0d2dec69f19d9c3a3940a401e3/KrovetzStemmer-0.7.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-build-lxw4loqy/KrovetzStemmer/setup.py", line 9, in <module>
        long_description = open('README.md').read()
    FileNotFoundError: [Errno 2] No such file or directory: 'README.md'
    
    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-build-lxw4loqy/KrovetzStemmer/
```

```
$ python -V
Python 3.6.9
```

**Comment by rueycheng** [comment]
Hi @setop, thanks for the feedback.

I've just uploaded a fix (0.8) to PyPI - can you let me know if that works for you?

**Comment by setop** [comment]
Yes it works now.
Many thanks.
